houston/generator/base.py

```diff
@@ -107,6 +107,11 @@ def failures(self):
         return self.__failures
 
 
+    @property
+    def coverage(self):
+        return self.__coverage
```

Comment:
What does it mean for a mission generator to have coverage?

Reply:
That's what I wanted to talk to you about. It means the same as mission generator to have `outcome`. I feel like the generator should only generate the missions and runner should be in charge of running and keeping track of results.

Reply:
I like your idea to tweak mission generator -- it makes much more sense. You could refactor it as a Python 3 generator. The only snag is that the guided mission generator actually uses the outcomes of previous executions to guide the search. (Still, Python 3 generators can also receive data, so we could tweak things to work).

Reply:
I'll refactor in another branch.